### PR TITLE
Update TokenOperationResponse to add Fiat Amounts

### PR DESCRIPTION
**Pull request by mohsinr**
Nexus API has Fiat amounts for transaction which are added to SDK now.

### CODE REVIEW

File: Nexus.Sdk.Token/Responses/TokenOperationResponse.cs [hunk 1]
```diff
@@ -64,5 +66,11 @@ public TokenOperationResponse(string code, string hash, OperationAccountResponse
 
         [JsonPropertyName("paymentReference")]
         public string PaymentReference { get; set; }
+
+        [JsonPropertyName("fiatAmount")]
+        public decimal FiatAmount { get; private set; }
```
Comment (nagarwal4) [inline Nexus.Sdk.Token/Responses/TokenOperationResponse.cs:71]: Make it optional

File: Nexus.Sdk.Token/Responses/TokenOperationResponse.cs [hunk 2]
```diff
@@ -64,5 +66,11 @@ public TokenOperationResponse(string code, string hash, OperationAccountResponse
...
+
+        [JsonPropertyName("netFiatAmount")]
+        public decimal NetFiatAmount { get; private set; }
```
Comment (nagarwal4) [inline Nexus.Sdk.Token/Responses/TokenOperationResponse.cs:74]: Make it optional

**Review by nagarwal4 — CHANGES_REQUESTED** [review]

File: Nexus.Sdk.Token/Responses/TokenOperationResponse.cs (hunk 2 above)
Comment (mohsinr) [reply Nexus.Sdk.Token/Responses/TokenOperationResponse.cs]: [Make fiat values optional in TokenOperationResponse](https://github.com/QuantozTechnology/Nexus.Sdk.Dotnet/pull/52/commits/fd7c22a7145780ca91153631128844dcddb72cc1)

File: Nexus.Sdk.Token/Responses/TokenOperationResponse.cs (hunk 1 above)
Comment (mohsinr) [reply Nexus.Sdk.Token/Responses/TokenOperationResponse.cs]: [Make fiat values optional in TokenOperationResponse](https://github.com/QuantozTechnology/Nexus.Sdk.Dotnet/pull/52/commits/fd7c22a7145780ca91153631128844dcddb72cc1)

File: Nexus.Sdk.Token/Responses/TokenOperationResponse.cs [hunk 3]
```diff
@@ -5,7 +5,7 @@ namespace Nexus.Sdk.Token.Responses
     public record TokenOperationResponse
     {
         [JsonConstructor]
-        public TokenOperationResponse(string code, string hash, OperationAccountResponses senderAccount, OperationAccountResponses receiverAccount, decimal amount, string created, string? finished, string status, string type, string? memo, string? message, string cryptoCode, string tokenCode, string paymentReference)
+        public TokenOperationResponse(string code, string hash, OperationAccountResponses senderAccount, OperationAccountResponses receiverAccount, decimal amount, string created, string? finished, string status, string type, string? memo, string? message, string cryptoCode, string tokenCode, string paymentReference, decimal fiatAmount, decimal netFiatAmount)
```
Comment (nagarwal4) [inline Nexus.Sdk.Token/Responses/TokenOperationResponse.cs:8]: Should be done here as well

Comment (mohsinr) [reply Nexus.Sdk.Token/Responses/TokenOperationResponse.cs]: Sorry for missing this one [Make fiat values optional](https://github.com/QuantozTechnology/Nexus.Sdk.Dotnet/pull/52/commits/ce68696e0d32efdadb57419cd8390fee9c34ca62)

**Review by nagarwal4 — APPROVED** [review]

**Pull request merged**